plugins/modules/xml.py

```diff
@@ -759,6 +761,7 @@ def child_to_element(module, child, in_type):
             (key, value) = next(iteritems(child))
             if isinstance(value, MutableMapping):
                 children = value.pop('_', None)
+                child_value = value.pop('_value', None)
```

Comment:
Hmm, this is a breaking change since if anyone ever set `_value` in the past, it suddenly does something else.

I'm not sure what's the best way to proceed here. Maybe this needs a module option which explicitly enables this behavior, so that users can opt-in to the new meaning of `_value` if they want.

Reply:
They essentially have the same problem, unless it's a character that isn't allowed in XML tags. According to https://www.w3.org/TR/2008/REC-xml-20081126/#sec-common-syn a tag name is of type `Name`, defined as:

```
[4]   	NameStartChar	   ::=   	":" | [A-Z] | "_" | [a-z] | [#xC0-#xD6] | [#xD8-#xF6] | [#xF8-#x2FF] | [#x370-#x37D] | [#x37F-#x1FFF] | [#x200C-#x200D] | [#x2070-#x218F] | [#x2C00-#x2FEF] | [#x3001-#xD7FF] | [#xF900-#xFDCF] | [#xFDF0-#xFFFD] | [#x10000-#xEFFFF]
[4a]   	NameChar	   ::=   	NameStartChar | "-" | "." | [0-9] | #xB7 | [#x0300-#x036F] | [#x203F-#x2040]
```

So `+` would definitely work. So how about using `+value` instead of `_value`?

Reply:
I think we can try this variant